.circleci/config.yml

```diff
@@ -598,6 +598,7 @@ jobs:
 
   static-analysis:
     executor: ndk-r22-latest-executor
+    resource_class: medium+
```

Comment:
Have been running out of memory specifically with the `Run android lint` job

https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/20403/workflows/9583319b-f5c5-44e1-a99d-85156f743878/jobs/102344